### ISSUE TITLE
Complete remaining steps to decommission was-thumbnail

### ISSUE THREAD

**Issue by andrewjbtw** [sul-dlss/was-thumbnail-service#211]
Decommissioning was-thumbnail was started in a proxy ticket in Argo https://github.com/sul-dlss/argo/issues/2091

Some of this work has been done already. Using this ticket to make a checklist of what's needed to finish the decommissioning process:

- [ ] Replace was-thumbnail-based embed viewer with viewer that makes use of the one thumbnail created during seed accessioning [done]
- [ ] Stop running the wasSeedDisseminationWF, which calls was-thumbnail-service (see was_robot [README](https://github.com/sul-dlss/was_robot_suite/blob/master/README.md#wasseeddissemination)
- [ ] Determine if any other workflow steps are connecting to was-thumbnail
- [ ] Check for any other was-thumbnail connections that might be made in the background (does SWAP expect some interaction with was-thumbnail?)
- [ ] Clean up Stacks to remove the thumbnails left there by was-thumbnail

**Comment by jcoyne** [comment]
@andrewjbtw can we accomplish the second step by editing the administrativeMetadata datastream in https://argo.stanford.edu/view/druid:xt299pt7593 and remove this clause?
```xml
  <dissemination>
    <workflow id="wasDisseminationWF"/>
  </dissemination>
```

**Comment by jcoyne** [comment]
I think the second step can be accomplished by removing this clause:
https://github.com/sul-dlss/was_robot_suite/blob/master/robots/was_dissemination/start_special_dissemination.rb#L18

**Comment by jmartin-sul** [comment]
the plan is for the first responder to shut down the server next week to see what breaks.  we'd prefer to just make it invisible to other services at first, without deleting the VM, in case we need to turn it back on.  but the plan is to make sure accessioning continues to work after the was-thumbnail VM is made unreachable for other services.

**Comment by mjgiarlo** [comment]
@andrewjbtw do you have a sense of what testing needs to take place for this? I wonder if we should do this during the Thursday 9am-12pm window when @tallenaz will be helping us test hydrus/lyberservices entanglement.

**Comment by andrewjbtw** [comment]
For the workflows, I think we just need to register and/or update a web archive seed object. It might be a good idea to accession a crawl object too since we're looking for hidden connections, but as far as I can tell the crawl objects aren't directly implicated in the thumbnails, just the seeds.

Other than that, I think we just need to check if there are any unexpected alerts or errors from SWAP. The thumbnail server checks SWAP to generate the thumbnails but I don't know if SWAP ever checks for the thumbnail server.

**Comment by mjgiarlo** [comment]
@andrewjbtw OK, how do you feel about doing this during the Thursday morning window? If you're open to it, I will reach out to Ops.

**Comment by andrewjbtw** [comment]
that works for me

**Comment by mjgiarlo** [comment]
Cc: @tallenaz, if you're open to it, here's another decommission test we can run on Thursday morning.

**Comment by tallenaz** [comment]
OK. To clarify, what's the second test? Is it "disassociate was-thumbnail from IP" the checklist above. If so, does "shut down was-thumbnail" count as an interpretation? Looks like there's an explicit "disassociate IP" function in Azure but no straightforward analogue in VMWare.

**Comment by mjgiarlo** [comment]
@tallenaz I think it'd be fine to shut down the VM, yes, but let me tag prior folks to see if they have any objections: @jcoyne @jmartin-sul @andrewjbtw

**Comment by andrewjbtw** [comment]
@tallenaz I think shutting it down should be fine.

**Comment by mjgiarlo** [comment]
@jcoyne @jmartin-sul 

I worked with @tallenaz and @andrewjbtw to decommission was-thumbnail-stage and -prod, and we can confirm that all the tests identified above worked just fine. No weird new behavior, and no unexpected exceptions in honeybadger. We will leave the thumbnail service boxes off but leave them around for three weeks, and then re-assess. That way if they are needed in a pinch, we can turn them back on.

@andrewjbtw will proceed with cleaning up stacks soon.

@tallenaz will check back on this in October and if none of us have lingering concerns, he will decommission the VMs for good. At that point, this issue can be closed.

(This is no longer being actively tracked by the @sul-dlss/infrastructure-team FR.)

AFAICT, the remaining was-thumbnail-service tendrils are in documentation and config:

* https://github.com/sul-dlss/shared_configs/tree/was-thumbnail-service-prod
* https://github.com/sul-dlss/shared_configs/tree/was-thumbnail-service-qa
* https://github.com/sul-dlss/shared_configs/tree/was-thumbnail-service-stage
* https://github.com/sul-dlss/was-thumbnail-service (deprecate/archive github repo?)
* https://github.com/sul-dlss/web-archiving/wiki/Glossary

**Comment by jmartin-sul** [comment]
> AFAICT, the remaining was-thumbnail-service tendrils are in documentation and config:
>  ...
>     * https://github.com/sul-dlss/was-thumbnail-service (deprecate/archive github repo?)

yeah, i'd be in favor of deprecating and archiving once we're sure the VM can be retired

**Comment by jmartin-sul** [comment]
note that in the unlikely event we have to resurrect this VM, we should revert these sdr-deploy and access-update-scripts changes:

https://github.com/sul-dlss/sdr-deploy/pull/36
https://github.com/sul-dlss/access-update-scripts/pull/122
